tests/integration-tests/tests/multiple_nics/test_multiple_nics/test_multiple_nics/run_instance_override.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+set -ex
+. "/etc/parallelcluster/cfnconfig"
+
+
+if [[ $cfn_node_type == "HeadNode" ]]; then
+    # Override run_instance attributes
+    cat > /opt/slurm/etc/pcluster/run_instances_overrides.json << 'EOF'
+{
+    "efa-enabled": {
```

Comment:
This cannot work as is because this must match the name of the queue and compute resource of the configuration file, so: 
efa-enabled --> queue-0
efa-enabled-i1 --> compute-resource-0

Reply:
copy-paste error, good catch thanks!